Source/Fuse.Models/Tests/Model.Test.uno

```diff
@@ -11,6 +11,53 @@ namespace Fuse.Models.Test
 {
 	public class ModelTest : ModelTestBase
 	{
+
+		[Test]
+		public void Args_Each()
+		{
+			var e = new UX.Model.Args.Each();
+			using (var root = TestRootPanel.CreateWithChild(e))
+			{
+				root.StepFrameJS();
+				Assert.AreEqual("(foo),(bar),(baz)", GetRecursiveText(e));
+				Assert.AreEqual(3, e.innerInstanceCount.Value);
+				Assert.AreEqual(1, e.outerInstanceCount.Value);
+			}
+		}
+
+		[Test]
+		public void Args_ScrollView()
+		{
+			var e = new UX.Model.Args.ScrollView();
+			using (var root = TestRootPanel.CreateWithChild(e))
```

Comment:
As this test requires a suitable height you should provide an explicit size to CreateWithChild.

Reply:
Good call. By the way, this test is really only there to check if we can call methods on a ScriptClass instance that was passed via an argument, so if you know of any nodes with a ScriptClass that would be better suited for this purpose, that'd be great :)